File: Loiane_Training_Java/Java_Basico/Exer Aula 14_15/Ex11.java
import java.util.Scanner;

class Ex11 {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		System.out.println("Digite o salario do colaborador");
		double salary = scan.nextDouble();

		System.out.println("O salario antes do reajuste: R$" + salary);

		int increasePercent = 0;
		double increaseValue = 0;
		if (salary <= 280) {
			increaseValue = (salary * 1.2) - salary;
			salary *= 1.2;
			increasePercent = 20;
		} else if (salary > 280 && salary < 700) {
			increaseValue = (salary * 1.15) - salary;
			salary *= 1.15;
			increasePercent = 15;
		} else if (salary > 700 && salary < 1500) {
			increaseValue = (salary * 1.1) - salary;
			salary *= 1.1;
			increasePercent = 10;
		} else {
			increaseValue = (salary * 1.05) - salary;
			salary *= 1.05;
			increasePercent = 5;
		}

		System.out.println("Percentual do reajuste: " + increasePercent + "%");
		System.out.println("Percentual de aumento aplicado: " + increaseValue);
		System.out.println("Valor do aumento: R$" + salary);

		scan.close();
	}
}

File: loiane-training-csharp/procedural-poo/c/Ex11.cs
using System;

public class Ex11
{
    static void Main(string[] args)
    {
        Console.WriteLine("Digite o salario do colaborador: \t");
        double Salary = double.Parse(Console.ReadLine());

        int Percent = 0;
        double IncreaseValue = 0;
        double NewSalary = 0;
        if (Salary <= 280)
        {
            Console.WriteLine("A");
            Percent = 20;
            NewSalary = Salary * (1 + (Percent / 100.0));
            IncreaseValue = NewSalary - Salary;
        }
        else if (Salary >= 280 && Salary < 700)
        {
            Percent = 15;
            Console.WriteLine("B");
            NewSalary = Salary * (1 + (Percent / 100.0)); ;
            IncreaseValue = NewSalary - Salary;
        }
        else if (Salary >= 700 && Salary < 1500)
        {
            Console.WriteLine("C");
            Percent = 10;
            NewSalary = Salary * (1 + (Percent / 100.0)); ;
            IncreaseValue = NewSalary - Salary;
        }
        else if (Salary >= 1500)
        {
            Console.WriteLine("D");
            Percent = 5;
            NewSalary = Salary * (1 + (Percent / 100.0)); ;
            IncreaseValue = NewSalary - Salary;
        }

        Console.WriteLine("Salario antes do reajuste:\t" + Salary);
        Console.WriteLine("Percentual aplicado:\t" + Percent);
        Console.WriteLine("Valor do aumento:\t" + IncreaseValue);
        Console.WriteLine("Novo salario apos aumento:\t" + NewSalary);
    }
}
